### PR TITLE
some simple doc updates

### PR DESCRIPTION
**Pull request by bradygaster**
There is a cue to Ctrl-F5 the app after opening it, but not after making changes to it. I added a comment that the reader should re-run the app to see the recent changes they made.

### CODE REVIEW

**Comment by bradygaster** [comment]
* added a few "ctrl-f5" lines prior to screen shots in case it wasn't implied to some attendees
* added a note to the readme to let them know they should clone the repo as the first mention was 3 clicks into the workshop, and it seemed like "back-click mitigation" might be good for continuity

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Blazor is an single-page app framework for building client-side web apps using .
 
 ## Getting Started
 
-Go ahead and [get started](/docs/00-get-started.md)!
+Go ahead and clone this repo to your machine, then dive t in and [get started](/docs/00-get-started.md)!
```
Comment (rynowak) [inline README.md]: `dive t` -> `dive`

File: docs/01-components-and-layout.md [hunk 2]
```diff
@@ -91,6 +91,8 @@ Once the component is initialized it will render its markup. Replace the markup
 </div>
 ```
 
+Run the app by hitting `Ctrl-F5`. Now you should see a list of the specials available.
```
Comment (rynowak) [inline docs/01-components-and-layout.md]: I don't super love putting VS-specific directions in if we're not going to cover VS Code as well. Possible to add?

Comment (bradygaster) [reply docs/01-components-and-layout.md]: I thought about that too, but, the majority of the screen shots involving the tooling also shows it, so I think we're safe. I can change this to "Run" as in most of the other places. Thanks.

Comment (rynowak) [reply docs/01-components-and-layout.md]: yeah that's a fair point.

**Pull request merged**